android-lib/build.gradle

```diff
@@ -54,19 +54,27 @@ android {
     abortOnError false
   }
   packagingOptions {
+    exclude 'META-INF/NOTICE'
+    exclude 'META-INF/LICENSE'
```

Comment:
@vinaygahlawat do we need to include the license files inside the package?

Reply:
@heyzooi @b-stolyarov I think that we should include the license files inside the package.

Reply:
Updated